src/string_input_popup.cpp

```diff
@@ -82,9 +86,15 @@ void string_input_popup::create_window()
 
     const int w_y = ( TERMY - w_height ) / 2;
     const int w_x = std::max( ( TERMX - w_width ) / 2, 0 );
-    w = catacurses::newwin( w_height, w_width, point( w_x, w_y ) );
-
     _starty = w_height - 2; // The ____ looks better at the bottom right when the title folds
+    w_full = catacurses::newwin( w_height, w_width, point( w_x, w_y ) );
+    if( !_description.empty() ) {
+        w_description = catacurses::newwin( description_height, w_width - 1, point( w_x,
+                                            w_y + 1 ) );
+        desc_view_ptr = std::make_unique<scrolling_text_view>( w_description );
+        description_view = desc_view_ptr.get();
```

Comment:
Is it necessary to store a raw pointer instead of using `desc_view_ptr` directly?

Reply:
I had initially set it up to match how `input_context` is handled in this class.  I have now moved to just using `desc_view_pointer` and everything seems to work correctly.
Does anyone know if there's a particular reason why `input_context` uses both a `unique_ptr` and a raw pointer here?

Reply:
I think it's because `input_context` can be provided by the caller, in which case it can only be stored as a raw pointer to the supplied input context.